Integrations/python/test/testParquetTools.py

```diff
@@ -86,6 +87,25 @@ def testCreation(self):
                 self.assertFalse(os.path.exists(fileLocation2))
         shutil.rmtree(baseDir)
 
+    def testDecimal(self):
+        jbigdecimal = jpy.get_type('java.math.BigDecimal')
+        table = dh.table_of([[jbigdecimal.valueOf(301, 2)],
+                                [jbigdecimal.valueOf(201,2)],
+                                [jbigdecimal.valueOf(101,2)]],
+                               [('decimal_value', dh.bigdecimal)])
+        self.assertIsNotNone(table)
+        baseDir = os.path.join(self.rootDir, "testCreation")
+        fileLocation = os.path.join(baseDir, 'table1.parquet')
+        if os.path.exists(fileLocation):
+            shutil.rmtree(fileLocation)
+        time.sleep(0.01)  # avoid race condition on file existence...
```

Comment:
It doesn't make any sense to me, unless python is deeply broken.  Filesystem operations, including stat related ones, are supposed to be atomic, or the universe would break badly (outside of NFS client side caching, but NFS v4 fixes that with locking, so that's that).  I removed the sleeps and ran the test 10 times without them and they pass, so, gone.